collect_app/src/main/java/org/odk/collect/android/preferences/AdminKeys.java

```diff
@@ -32,7 +32,7 @@
     public static final String KEY_MARK_AS_FINALIZED            = "mark_as_finalized";
 
     // server
-            static final String KEY_CHANGE_ADMIN_PASSWORD       = "admin_password";
+    static final String KEY_CHANGE_ADMIN_PASSWORD               = "admin_password";
```

Comment:
Did these actually fail the checkstyle check? The idea here is to line up the parts that are the same. That means the package private values are lined up on the `static` keyword.